address-validate-soap.postman_collection.json

```diff
@@ -0,0 +1,442 @@
+{
+	"variables": [],
+	"info": {
+		"name": "Address Validate SOAP",
+		"_postman_id": "64db4b4d-ccdc-46ce-4642-94a9d5d5fda0",
+		"description": "Address Validate API provides a SOAP interface for global address searching and validation. We currently support over 160 countries.",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "DoCanSearch",
+			"request": {
+				"url": {
+					"raw": "https://ws.ondemand.qas.com/ProOnDemand/V3/ProOnDemandService.asmx?WSDL",
+					"protocol": "https",
+					"host": [
+						"ws",
+						"ondemand",
+						"qas",
+						"com"
+					],
+					"path": [
+						"ProOnDemand",
+						"V3",
+						"ProOnDemandService.asmx"
+					],
+					"query": [
```

Comment:
Don't need the WSDL query param when invoking the service. Same for all the others.